src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+       action="/search"
+       method="get"
+       data-qa="big"
+       class="form"
+       >
+      <label
+         for="big icon">
+        <img
+           src="./images/Search.svg"
+           class="icon--cursor icon-big"
```

Comment:
this can be just `icon` and for small please add modifier class `icon icon--small`:

```suggestion
           class="icon"
```